experiments/opencog/pattern_matcher_vqa/pattern_matcher_vqa.py

```diff
@@ -305,7 +366,7 @@ def answerQuestion(self, record, use_pm=True):
                 self.logger.error('Question was not parsed')
                 return
             self.logger.debug('Scheme query: %s', queryInScheme)
-            answer = self.answerQuery(record.questionType, queryInScheme)
+            answer, bb_id, expr = self.answerQuery(record.questionType, queryInScheme)
```

Comment:
minor, but I would propose replace bb_id and expr by ```_``` if you don't need them it reads better